pipeline/console/environments.py

```diff
@@ -0,0 +1,239 @@
+import argparse
+import re
+from pathlib import Path
+from typing import List
+
+from pip._internal.commands.freeze import freeze
+from tabulate import tabulate
+
+from pipeline import PipelineCloud
+from pipeline.schemas.environment import (
+    EnvironmentCreate,
+    EnvironmentGet,
+    EnvironmentPatch,
+)
+from pipeline.schemas.pagination import Paginated
+from pipeline.util.logging import _print
+
+environment_re_pattern = re.compile(r"^[0-9a-zA-Z]+[0-9a-zA-Z\_\-]+[0-9a-zA-Z]+$")
+
+
+def _get_environment(name_or_id: str) -> EnvironmentGet:
+
+    remote_service = PipelineCloud(verbose=False)
+    remote_service.authenticate()
+
+    environment_information = EnvironmentGet.parse_obj(
+        remote_service._get(f"/v2/environments/{name_or_id}")
```

Comment:
As discussed in person, `by_name` is handled by a different endpoint, so we should fix that in a follow up PR.